test/integration/dialects/postgres/query-generator.test.js

```diff
@@ -870,7 +873,7 @@ if (dialect.match(/^postgres/)) {
           expectation: 'CREATE INDEX \"bar\" ON \"User\" (\"username\", \"isAdmin\")'
         }, {
           arguments: ['mySchema.User', ['username', 'isAdmin'], {}, 'User'],
```

Comment:
Hmm, is this a BC break @mickhansen - If the user passes tablename with schema as a string to addIndex it won't work. But we could argue that that was never the intended usage